docs/8-veba-troubleshooting.md

```diff
@@ -0,0 +1,232 @@
+
+```
+We now see that the Event Router came online, connected to vCenter, and successfully received an event.
+
+## OpenFaaS Function troubleshooting
+
+If a function is not behaving as expected, you can look at the logs to troubleshoot. First, SSH or console to VEBA as shown in the Requirements section.
+
+List out the pods:
+```
+kubectl get pods -A
+```
+
+This is the function output:
+```
+NAMESPACE        NAME                                   READY   STATUS      RESTARTS   AGE
+kube-system      coredns-584795fc57-4bp2s               1/1     Running     1          6d4h
+kube-system      coredns-584795fc57-76pwr               1/1     Running     1          6d4h
+kube-system      etcd-veba01                            1/1     Running     2          6d4h
+kube-system      kube-apiserver-veba01                  1/1     Running     2          6d4h
+kube-system      kube-controller-manager-veba01         1/1     Running     3          6d4h
+kube-system      kube-proxy-fvf2n                       1/1     Running     2          6d4h
+kube-system      kube-scheduler-veba01                  1/1     Running     2          6d4h
+kube-system      weave-net-v9jss                        2/2     Running     6          6d4h
+openfaas-fn      powercli-entermaint-d84fd8d85-sjdgl    1/1     Running     1          6d4h
+openfaas         alertmanager-58f8d787d9-nqwm8          1/1     Running     1          6d4h
+openfaas         basic-auth-plugin-dd49cd66b-rv6n7      1/1     Running     1          6d4h
+openfaas         faas-idler-59ff9778fd-84szz            1/1     Running     4          6d4h
+openfaas         gateway-74f6f9489b-btgz8               2/2     Running     5          6d4h
+openfaas         nats-6dfbf45d77-9swph                  1/1     Running     1          6d4h
+openfaas         prometheus-5f5494b54f-srs2d            1/1     Running     1          6d4h
+openfaas         queue-worker-59b67bf4-wqhm5            1/1     Running     4          6d4h
+projectcontour   contour-5cddfc8f6-hpzn8                1/1     Running     1          6d4h
+projectcontour   contour-5cddfc8f6-tdv2r                1/1     Running     2          6d4h
+projectcontour   contour-certgen-wrgnb                  0/1     Completed   0          6d4h
+projectcontour   envoy-8mdhb                            1/1     Running     2          6d4h
+vmware           tinywww-7fcfc6fb94-v7ncj               1/1     Running     1          6d4h
+vmware           vmware-event-router-5dd9c8f858-9g44h   1/1     Running     4          6d4h
+```
+
+First, we want to see if the event router is capturing events and forwarding them on to a function. 
+
+Use this command to follow the live Event Router log
+```
+kubectl logs -n vmware vmware-event-router-5dd9c8f858-9g44h --follow
```

Comment:
these `kubectl` commands don't require the `kubeconfig` flag to work?

Reply:
@embano1 Nope, the login user already has the kubeconfig context, so no need to specify :)